arm/Microsoft.ServiceBus/namespaces/version.json

```
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.4"
+    "version": "0.6"
```

Good suggestion, its a new functionality, however it's enough with one bump. ;)

```suggestion
    "version": "0.5"
```